src/javascript/app/pages/user/account/settings/financial_assessment.js

```diff
@@ -44,6 +45,12 @@ const FinancialAssessment = (() => {
             });
         }
 
+        if (LocalStore.get('financial_assessment_redirect')) {
+            const is_mt5_financial = /^(real_vanuatu_standard)$/.test(LocalStore.get('mt5_account'));
```

Comment:
couldn't you get this from `Client.get('mt5_account')`?

Reply:
yes i could, but we need to remove this check if the user navigates away and go back to the page via Settings menu. we need to show Trading Experience form only when the user go the Financial Assessment page from MT5 new account page

Reply:
I see! thanks

Reply:
but isn't that covered by the `financial_assessment_redirect`? let's talk in person 😅